### PR TITLE
missing word 'you' in test_network.md

### PR DESCRIPTION
**Pull request by fatcat22**
#### Type of change

- Documentation update

#### Description

A simple fix with missing a word in test_network.md

### CODE REVIEW

**Comment by joealewine** [comment]
Thank you for finding this error and submitting a fix!

However, you need to amend this commit with a proper signoff before it can be merged. Click the "details" link to the right of the DCO check and follow the instructions.

**Comment by fatcat22** [comment]
Sorry for broken the rule, I have fixed it.

**Review by joealewine — APPROVED** [review]

**Pull request merged**
